ipamd/ipamd.go

```diff
@@ -63,6 +66,12 @@ var (
 			Help: "The number of maximum ENIs can be attached to the instance",
 		},
 	)
+	ipMax = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "ip_max",
+			Help: "The number of maximum IPs can be allocated to the instance",
```

Comment:
s/number of maximum IPs can/maximum number of IP addresses that can/

Reply:
please correct as advised.  this is user facing description, let's get the grammar right.